From 0cd3f561efa9adce840140720e0581355db3e554 Mon Sep 17 00:00:00 2001
From: Zou Wei <zou_wei@huawei.com>
Date: Tue, 8 Dec 2020 20:28:09 +0800
Subject: [PATCH] platform/x86: ISST: Mark mmio_range_devid_0 and
 mmio_range_devid_1 with static keyword
Git-commit: 0cd3f561efa9adce840140720e0581355db3e554
Patch-mainline: v5.11-rc1
References: bsn#314
Modified-by-SEL: No


Fix the following sparse warnings:

drivers/platform/x86/intel_speed_select_if/isst_if_mmio.c:23:24: warning: symbol 'mmio_range_devid_0' was not declared. Should it be static?
drivers/platform/x86/intel_speed_select_if/isst_if_mmio.c:28:24: warning: symbol 'mmio_range_devid_1' was not declared. Should it be static?

Signed-off-by: Zou Wei <zou_wei@huawei.com>
Link: https://lore.kernel.org/r/1607430489-116200-1-git-send-email-zou_wei@huawei.com
Signed-off-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 drivers/platform/x86/intel_speed_select_if/isst_if_mmio.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/platform/x86/intel_speed_select_if/isst_if_mmio.c b/drivers/platform/x86/intel_speed_select_if/isst_if_mmio.c
index 2906cfee5d9c..ff49025ec085 100644
--- a/drivers/platform/x86/intel_speed_select_if/isst_if_mmio.c
+++ b/drivers/platform/x86/intel_speed_select_if/isst_if_mmio.c
@@ -20,12 +20,12 @@ struct isst_mmio_range {
 	int end;
 };
 
-struct isst_mmio_range mmio_range_devid_0[] = {
+static struct isst_mmio_range mmio_range_devid_0[] = {
 	{0x04, 0x14},
 	{0x20, 0xD0},
 };
 
-struct isst_mmio_range mmio_range_devid_1[] = {
+static struct isst_mmio_range mmio_range_devid_1[] = {
 	{0x04, 0x14},
 	{0x20, 0x11C},
 };
-- 
2.31.1

